src/main/java/discord4j/discordjson/json/ChannelCreateRequest.java

```diff
@@ -18,17 +18,47 @@ static ImmutableChannelCreateRequest.Builder builder() {
     }
 
     String name();
+
     Possible<Integer> type();
+
     Possible<String> topic();
+
     Possible<Integer> bitrate();
+
     @JsonProperty("user_limit")
     Possible<Integer> userLimit();
+
     @JsonProperty("rate_limit_per_user")
     Possible<Integer> rateLimitPerUser();
+
     Possible<Integer> position();
+
     @JsonProperty("permission_overwrites")
     Possible<List<OverwriteData>> permissionOverwrites();
+
     @JsonProperty("parent_id")
     Possible<String> parentId();
+
     Possible<Boolean> nsfw();
+
+    @JsonProperty("default_thread_rate_limit_per_user")
+    Possible<Integer> defaultThreadRateLimitPerUser();
+
+    Possible<Integer> flags();
+
+    @JsonProperty("default_auto_archive_duration")
+    Possible<Integer> defaultAutoArchiveDuration();
+
+    @JsonProperty("default_reaction_emoji")
+    Possible<DefaultReactionData> defaultReactionEmoji();
```

Comment:
Based in docs this can be Possible and Optional.
```suggestion
    Possible<Optional<DefaultReactionData>> defaultReactionEmoji();
```

Reply:
Fixed